gpflow/models/training_interface.py

```diff
@@ -12,25 +12,25 @@
 Data = TypeVar("Data", RegressionData, InputData)
 
 
-class InternalDataTrainingInterface:
+class InternalDataTrainingLossMixin:
     def training_loss(self):
         """
         Specify a training loss for this model
         """
-        raise NotImplementedError
+        return -self.maximum_a_posteriori_objective(data)
```

Comment:
I presume you mean
```suggestion
        return -self.maximum_a_posteriori_objective()
```
?